pkg/kubelet/dockershim/docker_container.go

```diff
@@ -438,3 +447,20 @@ func (ds *dockerService) UpdateContainerResources(_ context.Context, r *runtimea
 	}
 	return &runtimeapi.UpdateContainerResourcesResponse{}, nil
 }
+
+func (ds *dockerService) performPlatformSpecificContainerCleanupForContainer(containerID string) {
```

Comment:
How about calling this `performPlatformSpecificCleanupForContainer`?

Reply:
👍